modin/config/pubsub.py

```diff
@@ -75,6 +75,22 @@ class ExactStr(str):
         or (isinstance(value, str) and value.strip().isdigit()),
         help="an integer value",
     ),
+    dict: TypeDescriptor(
+        decode=lambda value: {
+            key.strip(): int(val) if val.strip().isdigit() else val.strip()
```

Comment:
```suggestion
            key.strip(): int(val.strip()) if val.strip().isdigit() else val.strip()
```
otherwise what's the point of checking `val.strip().isdigit()`?